kafka/README.md

```diff
@@ -1,32 +1,437 @@
 # Kafka Integration
 
-## Overview
+# Overview
 
-Get metrics from kafka service in real time to:
+Connect Kafka to Datadog in order to:
 
-* Visualize and monitor kafka states
-* Be notified about kafka failovers and events.
+* Visualize the performance of your cluster in real time
+* Correlate the performance of Kafka with the rest of your applications
 
-## Installation
+This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page. You can specify the metrics you are interested in by editing the configuration below. To learn how to customize the metrics to collect visit the [JMX Checks documentation](/integrations/java) for more detailed instructions.
 
-Install the `dd-check-kafka` package manually or with your favorite configuration manager
+To collect Kafka consumer metrics, see the kafka_consumer check.
 
-## Configuration
+# Installation
 
-Edit the `kafka.yaml` file to point to your server and port, set the masters to monitor
+The Agent's Kafka check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Kafka nodes. If you need the newest version of the check, install the `dd-check-kafka` package.
```

Comment:
@truthbk since jmx checks are mostly just yml files how do those get distributed. Is it only as part of new agent releases? I agree having them as a package likely doesn't make sense.